src/FakeItEasy/Captured.of.T.cs

```diff
@@ -0,0 +1,24 @@
+namespace FakeItEasy
+{
+    using System;
+
+    /// <summary>
+    /// Holds argument values captured from calls to a faked member.
+    /// </summary>
+    /// <typeparam name="T">The type of the argument.</typeparam>
+    public class Captured<T> : Captured<T, T>

```

Comment:
nit: `Captured<TArgument>`, for consistency with `Captured<TArgument, TCapture>`